sdk/formrecognizer/Azure.AI.FormRecognizer/tests/FormRecognizerClient/FormRecognizerClientLiveTests.cs

```diff
@@ -895,8 +895,8 @@ public async Task StartRecognizeReceiptsCanParseMultipageFormWithBlankPage()
 
                 if (formIndex == 0 || formIndex == 2)
                 {
-                    var sampleField = recognizedForm.Fields["MerchantName"];
-                    var expectedValueData = formIndex == 0 ? "Bilbo Baggins" : "Frodo Baggins";
+                    var sampleField = recognizedForm.Fields["Total"];
+                    var expectedValueData = formIndex == 0 ? "430.00" : "4300.00";
```

Comment:
Same reasoning here: we're using the receipt prebuilt model to recognize invoices. We stopped getting the `MerchantName` back in the last page, so we're using another parameter to check page order.